Utils/Elasticsearch/requests/hashtag-steps-statistics-q.json

```diff
@@ -38,9 +36,21 @@ GET <index>/task/_search
         "cpu": {
           "sum": {"field": "hs06"}
         },
-        "walltime": {
-         "sum": {"script": {"inline":"doc['end_time'].value - doc['start_time'].value"}}
-        },
+		"timestamp_defined": {
+			"filter": {
+             "bool": {
```

Comment:
Just a comment: I believe it would also work without `"bool": {`, simply with its content: `"must": [...]`.

Not just a comment: indentation is broken. For such multi-level JSON it is very important for readability to keep it aligned properly, or else it is too easy to lose the context.

Reply:
@mgolosova 
I tried to check and, unfortunately, it doesn't work. :(
It seems that 'bool' is a mandatory part of this construction.